# Nessuna richiesta
---
float32 avg_linear
float32 avg_angular